App/iOS/Delegates/AppDelegate.swift

```diff
@@ -52,6 +52,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
       }
     }
+    switches[.rewardsFlags] = BraveRewards.Configuration.current().flags
```

Comment:
Need @kylehickinson to approve this line

Reply:
@iccub These are my own changes lol